templates/app/angular-default/template/.vscode/settings.json

```
@@ -0,0 +1,20 @@
+{
+	"files.exclude": {
+		"build/": true,
+		"i18n/": true,
+		"platform/": true,
+		"Resources/": true
```

These changed with the latest Angular template. I don't touch `i18n` and `platform` anymore so they basically work like in a classic app.

```suggestion
		"build/": true,
		"Resources/": true
```